solutions/secure-hybrid-network/nestedtemplates/azure-network-azuredeploy.bicep

```diff
@@ -386,15 +384,19 @@ resource spokeNetwork_name_Microsoft_Insights_default_logAnalyticsWorkspace 'Mic
   }
 }
 
-resource vpnGateway_publicIPAddress 'Microsoft.Network/publicIPAddresses@2023-04-01' = if (configureSitetosite) {
+resource vpnGateway_publicIPAddress 'Microsoft.Network/publicIPAddresses@2023-11-01' = if (configureSitetosite) {
   name: vpnGateway.publicIPAddressName
   location: location
+  sku: {
+    name: 'Standard'
+    tier: 'Regional'
+  }
   properties: {
-    publicIPAllocationMethod: 'Dynamic'
+    publicIPAllocationMethod: 'Static'
```

Comment:
Standard sku publicIp /subscriptions/9c257b18-344d-4d4b-b09e-5c0a80b50e48/resourceGroups/rg-site-to-site-mock-prem-eastus2/providers/Microsoft.Network/publicIPAddresses/pip-onprem-vpn-gateway must have AllocationMethod set to Static. (Code: StandardSkuPublicIPAddressesMustBeStatic)